angular/src/assets/i18n/fr.json

```diff
@@ -1,12 +1,12 @@
 {
   "en": "English",
-  "no": "Norsk",
+  "no": "Norwegian",
```

Comment:
This should be "Norsk". These are local names, if you are Russian or Norwegian, you want to see your local language here.

Reply:
![image](https://github.com/block-core/blockcore-wallet/assets/6504337/0f2a33f8-6d4f-4775-9261-b9f124382d4e)